bapc/ingress.yaml

```diff
@@ -0,0 +1,40 @@
+apiVersion: networking.k8s.io/v1beta1
+kind: Ingress
+metadata:
+  name: bapc
+  namespace: default
+spec:
+  rules:
+    - host: bapc.eu
+      http:
+        paths:
+        - backend:
+            serviceName: bapc-2020
+            servicePort: 80
+          path: /2020
+  tls:
+    - hosts:
+        - bapc.eu
+      secretName: ingress-ext-tls
+---
+apiVersion: networking.k8s.io/v1beta1
+kind: Ingress
+metadata:
+  annotations:
+    nginx.ingress.kubernetes.io/temporal-redirect: https://2020.bapc.eu
+    nginx.ingress.kubernetes.io/use-regex: "true"
+  name: bapc-redirect
+  namespace: default
+spec:
+  rules:
+    - host: bapc.eu
+      http:
+        paths:
+          - path: /$
+            backend:
+              serviceName: bapc.eu
+              servicePort: 80
+  tls:
+    - hosts:
+        - bapc.eu
+      secretName: ingress-ext-tls
```

Comment:
@praseodym Ik weet niet of dit klopt aangezien het niet in een `x.ch.tudelft.nl`-formaat is. Werkt dit ook voor `bapc.eu` of is het slim om iets van een `bapc.ch.tudelft.nl` te hebben voor nu?

Reply:
De domeinnaam moet volgens mij `2020.bapc.eu` worden?